new/components/restservice/scripts/create.sh

```diff
@@ -17,10 +17,11 @@ export AGENT_SOURCE_URL=$(ctx node properties agent_module_source_url)
 # TODO: change to /opt/cloudify-rest-service
 export REST_SERVICE_HOME="/opt/manager"
 export REST_SERVICE_VIRTUALENV="${REST_SERVICE_HOME}/env"
-# guni.conf currently contains localhost for all endpoints. We need to change that.
+# cloudify-rest.conf currently contains localhost for all endpoints. We need to change that.
 # Also, MANAGER_REST_CONFIG_PATH is mandatory since the manager's code reads this env var. it should be renamed to REST_SERVICE_CONFIG_PATH.
-export MANAGER_REST_CONFIG_PATH="${REST_SERVICE_HOME}/guni.conf"
-export REST_SERVICE_CONFIG_PATH="${REST_SERVICE_HOME}/guni.conf"
+export MANAGER_REST_CONFIG_PATH="${REST_SERVICE_HOME}/cloudify-rest.conf"
+export REST_SERVICE_CONFIG_PATH="${REST_SERVICE_HOME}/cloudify-rest.conf"
+export MANAGER_REST_SECURITY_CONFIG_PATH="${REST_SERVICE_HOME}/rest-security.conf"
```

Comment:
can we call this cloudify-rest-security.conf just to correspond with the name of the other config? :)